plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/AwsSdkClient.kt

```diff
@@ -24,7 +24,12 @@ class AwsSdkClient : SdkClientProvider, Disposable {
     private val sdkHttpClient: SdkHttpClient by lazy {
         LOG.info { "Create new Apache client" }
         val httpClientBuilder = ApacheHttpClient.builder()
-            .proxyConfiguration(ProxyConfiguration.builder().useSystemPropertyValues(false).build())
+            .proxyConfiguration(
+                ProxyConfiguration.builder()
+                    .useSystemPropertyValues(false)
+                    .useEnvironmentVariableValues(false)
```

Comment:
test case would be nice too

Reply:
Added tests that assert the client functions as expected when system property or env var proxy config is passed.